stack-cmbm/recipes/deploy-nginxapp.rb

```diff
@@ -0,0 +1,101 @@
+include_recipe 'nginx-app::server'
```

Comment:
Granted. There is a major indention bug that very probably is somewhat confusing to the reader.